spec/features/submit_feedback_spec.rb

```diff
@@ -1,26 +1,58 @@
 require 'rails_helper'
 
 RSpec.feature 'Submit feedback', js: true do
-  before do skip 'Features specs not yet fixed' end
-
   include FeaturesHelper
 
-  scenario 'submitting feedback' do
+  normalised_body = lambda do |r1, r2|
```

Comment:
The body contains two fields which can vary:

* `user_agent`: This differs between local machines and Circle CI
* `referrer`: The port changes from run to run

The `normalised_body` matcher truncates these fields.